### PR TITLE
Servable adds mobilenetv2 support

### PR DESCRIPTION
**Pull request by hellowaywewe**

### CODE REVIEW

File: tinyms/serving/servable/servable.py [hunk 1]
```diff
@@ -62,12 +63,14 @@ def predict(instance, servable_name, servable_model):
     # parse the input data
     input = ts.array(json.loads(instance['data']), dtype=instance['dtype'])
     # build the network
-    net = lenet5(class_num=class_num) if model_name == "lenet5" else resnet50(class_num=class_num)
+    net_func = globals().get(model_name)
```
Comment (leonwanghui) [inline tinyms/serving/servable/servable.py:66]: Where is `globals()` defined?

File: tinyms/serving/servable/servable.py [hunk 2]
```diff
@@ -62,12 +63,14 @@ def predict(instance, servable_name, servable_model):
...
+    if net_func is not None:
```
Comment (leonwanghui) [inline tinyms/serving/servable/servable.py:67]: This should be changed to:
```python
if net_func is None:
    err_msg = "The net function is None!"
    return {"status": 1, "err_msg": err_msg}
net = net_func(class_num=class_num)
```

**Comment by hellowaywewe** [comment]
@leonwanghui thank you for your advice, I have updated it, Plz review it again, Thanks.

**Review by leonwanghui — APPROVED** [review]
LGTM

**Review by huxiaoman7 — APPROVED** [review]

**Pull request merged**